examples/lines_bars_and_markers/stairs_demo.py

```diff
@@ -43,6 +43,15 @@
     ax.legend()
 plt.show()
 
+#############################################################################
+# *baseline* can take an array to allow for stacked histogram plots
+
+A = np.tile([1, 2, 3], (3, 1)) * np.arange(1, 4).reshape(-1, 1)
+A = np.vstack([np.zeros(3), A])
```

Comment:
That's pretty arcane. Just go explicit:

```suggestion
A = [[0, 0, 0],
     [1, 2, 3],
     [2, 4, 6],
     [3, 6, 9]]
```
I'd even favor some more natural looking data. The monotonic steps look somewhat artificial.